apps/mobile/src/components/headers/animated-header/animated-header-screen.layout.tsx

```diff
@@ -23,6 +23,7 @@ interface AnimatedHeaderScreenLayoutProps extends HasChildren {
   subtitle?: string | React.ReactNode;
   contentContainerStyles?: ScrollViewStylesProps;
   isHeaderReversible?: boolean;
+  scrollable?: boolean;
```

Comment:
@tigranpetrossian I tried to get this `AnimatedHeader` working with `FlatList` and it does work however it gives a warning about nested virtualized lists (rightly). 

https://github.com/user-attachments/assets/22b93972-cebf-4dae-ac23-02ff6ef6b639

I refactored this component a bit to enable `scrollable` or not but maybe I should just throw that away and revert to a fixed header for `activity` ?